### DISCUSSION TITLE
production installation fails

### DISCUSSION THREAD

**Discussion by alevillani** [oqtane/oqtane.framework#2694] [General]
Hello
someone can help me? I follow all detailed instrtuctions on a personal pc and on a public iis server (please consider I'm using DNN from 17 years, I know iis, db, setup etc) but in all my tests I will receive a redirect to https and a page blank, the first install setup doesn't start in any way.

**Comment by alevillani** [comment]
please consider that in production I solved, added in hosts file the correct dns name used that wan not resolved....
but on a siple laptop the hosts file is correctly set 
127.0.0.1 oqtane.dev 

no firewall enabled

but nothing!

**Reply by thabaum** [reply]
The IIS site and virtual folder setup to resolve Oqtane.dev and you are able to pull up an html "hello world" basic test page in the home root folder on the laptop?  Fix that issue first then you can troubleshoot Oqtane as needed.  Most load up Oqtane with Visual Studio to run it locally.  It has IIS built in.  I will setup a few test runs from the instructions provided to see if I can notice anything that may use some clarity or updated for me to run it.  I should have some more thoughts on this if you are still having the issue possibly tomorrow or later this week if I can make time.
